cypress/support/commands.js

```diff
@@ -62,13 +62,13 @@ Cypress.Commands.add('idapiMockPattern', (status, body, pattern) => {
   cy.request(getMockOptions(status, body));
 });
 
-Cypress.Commands.add('idapiLastPayloadIs', (expected) => {
+Cypress.Commands.add('lastPayloadIs', (expected) => {
```

Comment:
Are we using this command, maybe we can remove it?

Reply:
it's being used in the onboarding flow spec (may need to expand the diff in github)